apps/studio/src/features/editing-experience/components/ScheduledPublishingModal/ScheduledPublishingModal.tsx

```diff
@@ -0,0 +1,427 @@
+              const valueFormatted = format(date, "HH:mm")
+              return (
+                <Badge
+                  key={displayFormatted}
+                  variant="outline"
+                  cursor="pointer"
+                  borderWidth="1px"
+                  borderColor="blue.200"
+                  onClick={() => setValue("publishTime", valueFormatted)}
+                >
+                  <Text textStyle="legal" color="interaction.main.default">
+                    {displayFormatted}
+                  </Text>
+                </Badge>
+              )
+            })}
+          </HStack>
+        </VStack>
+      )}
+    </>
+  )
+}
+
+const PublishModeCard = ({
+  icon,
+  title,
+  description,
+  isSelected,
+  onSelect,
+}: {
+  icon: IconType
+  title: string
+  description: string
+  isSelected: boolean
+  onSelect: () => void
+}) => {
+  return (
+    <HStack
+      display="flex"
+      alignItems="flex-start"
+      border="1px"
+      borderColor={isSelected ? "base.divider.brand" : "base.divider.medium"}
+      boxShadow={isSelected ? "sm" : undefined}
+      padding="0.75rem"
+      spacing="0.5rem"
+      cursor="pointer"
+      borderRadius="md"
+      onClick={onSelect}
+    >
+      <Icon
+        as={icon}
+        boxSize="1.5rem"
+        py="0.125rem"
+        color={isSelected ? "base.content.brand" : "base.content.strong"}
+      />
+      <VStack align="stretch" spacing={0}>
+        <HStack display="flex" spacing="0" alignItems="center">
+          <Text
+            textStyle="subhead-2"
+            color={isSelected ? "base.content.brand" : "base.content.strong"}
+          >
+            {title}
+          </Text>
+          {isSelected && (
+            <Icon
+              as={BiCheck}
+              boxSize="1rem"
+              color="utility.feedback.success"
+            />
+          )}
+        </HStack>
+        <Text textStyle="body-2">{description}</Text>
+      </VStack>
+    </HStack>
+  )
+}
+
+const SchedulePublishBanner = () => {
+  const { getValues } =
+    useFormContext<z.input<typeof schedulePublishClientSchema>>()
+
+  return (
+    <HStack
+      spacing="0.5rem"
+      alignItems="flex-start"
+      bgColor="utility.feedback.info-subtle"
+      borderRadius="0.25rem"
+      p="0.75rem"
+    >
+      <Icon as={BiCalendarCheck} boxSize="1rem" color="base.content.default" />
+      <Text textStyle="body-2" color="base.content.strong" display="inline">
+        We will publish this page at{" "}
+        <Text display="inline" textStyle="subhead-2">
+          {format(
+            parse(getValues("publishTime"), "HH:mm", new Date()),
+            "hh:mm a",
+          )}
+        </Text>
+        , Singapore Standard Time, on{" "}
+        <Text display="inline" textStyle="subhead-2">
+          {format(getValues("publishDate"), "MMMM d, yyyy")}
+        </Text>
+        . Changes will be live on your site approximately 15-30 minutes after
```

Comment:
this seems inconsistent with the 5-10mins timing stated in "publish now" - any reason? since they should be similar